### ISSUE TITLE
Show Name instead of Username

### ISSUE THREAD

**Issue by amonemian** [RocketChat/Rocket.Chat#9235]
Is it possible to show Name instead of Usernames in Direct Message list?

**Comment by amonemian** [comment]
Administration-->layout-->User Interface --> Use Real Name

Solved!

**Comment by Nepoxx** [comment]
I'm having the same issue as @tnatanael 

For some users I'm seeing their usernames, even if their full name is correctly entered (I also see their full name if I look at their profile or the admin user list). I've tried turning the option on and off, to no avail.

**Comment by piotrkochan** [comment]
Same situation here, some users still have theirs usernames instead of names.

**Comment by oryfoxer7410** [comment]
Display name issue is not resolved for 0.69.2 - Can confirm issue still exists with LDAP users. Some work, others do not.

**Comment by donkelley** [comment]
agreed.  It seems random to me - most users show username, only a couple show name, even though I added their names personally as admin.  Is it only working if THEY add their own name?  or the other way around?  (as opposed to the admin doing it)?

**Comment by ArchimedesFM** [comment]
Nobody changes anything except the avatar. All users are created by LDAP sync.
In my navigation pane (left chatpartner list) only a couple of user show as username; most users are display with the name. The same user (shown with his name in my chatpartner list) can show with his username in chatpartner list of my colleague. In our environment the name is the default displayname; not the username.

**Comment by cysk** [comment]
Can confirm, that it happens when new ldap-users enter the system. We authenticate against MS AD via LDAPS and in Rocket.Chat Version 0.68.4 the issue is still present. It has been working in the past. Entering "Administration -> Users -> Some User" the full name is shown.

**Comment by joseluisq** [comment]
It doesn't work for my users created by LDAP sync.

**Comment by iesit** [comment]
> 
> 
> It doesn't work for my users created by LDAP sync.

I've been working with MS AD LDAP and Rocket (0.74.3) a bunch lately, and I've found that all users I've imported are showing the correct Name and Username in the Admin -> Users list.  I am using "sAMAccountName" for Admin -> LDAP -> Sync / Import -> Username Field and {"cn":"name", "mail":"email"} for the User Data Field Map.  May want to ensure that all of your LDAP users have the correct CN set?  In MS AD, I did some troubleshooting of the LDAP query that I am using by launching Powershell with Active Directory Module and running: 

`([ADSISearcher]"(&(&(&(objectCategory=person)(mail=*)(objectClass=user)(!(userAccountControl:1.2.840.113556.1.4.803:=2))(!(mail=postmaster@mydomain.com)))))").FindAll() | Select Path`

Which gave me an idea of which users would be returned... I believe the CN is listed in results of this query.
